src/components/arkfanePanel/ArkfanePanel.tsx

```diff
@@ -37,16 +70,27 @@ const ArkfanePanel: React.FC<Props> = (props) => {
     }, [valgtFane]);
 
     return (
-        <StyledPanel>
+        <StyledPanel error={+hasError}>
+            {hasError && <StyledErrorColored />}
```

Comment:
bør gjøre ikonet accessible, med en prop title="error" (og aria-hidden når det ikke trenger å vises for skjermleser, men errors er vel greit å utheve)

Reply:
Skal ordne det

Reply:
👍 men jeg prøvde å si at vi ikke skulle ha aria-hidden på denne

Reply:
Fordi skjermleser trenger å vite at det er error

Reply:
Beklager, leste helt feilt.